src/brineconnector/utils.py

```diff
@@ -22,7 +25,7 @@ def sign_msg_hash(nonce: CreateOrderNoncePayload, private_key: str, option: Lite
 
 
 def create_user_signature(private_key: str, option: Literal['mainnet', 'testnet'] = 'mainnet') -> str:
```

Comment:
Please remove this function, to prevent stark key mismatch.

Reply:
I'm keeping this method for current use and local testing, was thinking to remove this once I start to work with nodejs one completely